docs/meshstack.azure.landing-zones.md

```diff
@@ -32,17 +32,21 @@ Blueprints must reside inside a Management Group. It is assumed it is in the sam
 
 Operators must ensure to create these management groups in the meshcloud AAD Tenant before configuring them for use in a meshLanding Zone.
 
-The following parameter can be used in the Blueprint:
+#### Available Blueprint Parameters
 
-| Parameter          | Description                                                                                |
-|--------------------|:-------------------------------------------------------------------------------------------|
-| customerIdentifier | Customer Identifier                                                                        |
-| ~~costcenter~~     | ID of the CostCenter defined for this meshProject. (Deprecated. Please use tagCostCenter)  |
-| projectIdentifier  | The project identifier                                                                     |
-| subscriptionId     | The ID of the Azure Subscription associated with this meshProject                          |
+The following parameter can be used in the Blueprint:
 
-In addition, any payment settings, project tags or customer tags can also be used in the Blueprints. These parameter keys will have the prefix `tag`.
-For example, the value of the tag `costCenter` will be made available via the key `tagCostCenter`.
+| Parameter          | Description                                                                               |
+|--------------------|:------------------------------------------------------------------------------------------|
+| customerIdentifier | Customer Identifier                                                                       |
+| ~~costcenter~~     | ID of the CostCenter defined for this meshProject. (Deprecated. Please use tagCostCenter) |
+| projectIdentifier  | The project identifier                                                                    |
+| subscriptionId     | The ID of the Azure Subscription associated with this meshProject                         |
+| tagCostCenter      | Example for a  [MetaData Tag](./meshstack.tag-schema.md) named `costCenter`               |
+
+As the example `tagCostCenter` in the above table indicates, any payment settings, project tags or customer tags can also be used in the Blueprints. These parameter keys will have the prefix `tag`.
```

Comment:
d: check if they not only have prefix, I think there's also a "name normalization" rule that we should explain (i.e. prepend tag, first letter is uppercase)